src/mono/wasm/runtime/exports.ts

```diff
@@ -134,9 +134,9 @@ let exportedAPI: DotnetPublicAPI;
 // it exports methods to global objects MONO, BINDING and Module in backward compatible way
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 function initializeImportsAndExports(
-    imports: { isGlobal: boolean, isNode: boolean, isShell: boolean, isWeb: boolean, locateFile: Function },
+    imports: { isES6: boolean, isGlobal: boolean, isNode: boolean, isShell: boolean, isWeb: boolean, locateFile: Function, quit_: Function },
```

Comment:
The `isES6` is for some future use, isn't it?

Reply:
I forgot to do it, thanks! I would be changing default for disableDotnet6Compatibility